README.md

```diff
@@ -18,42 +18,49 @@ The design decisions we made to achieve these goals will help inform policy make
 
 # Architecture
+
 We have explored several architectures under two broad categories as follows:
+
 ## UHS-Based Transaction Processor
+
 We explored two system architectures for transaction settlement based on an [unspent transaction output (UTXO)](https://en.wikipedia.org/wiki/Unspent_transaction_output) data model and transaction format.
 Both architectures implement the same schema representing an [unspent hash set (UHS)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-May/015967.html) abstraction.
 One architecture provides [linearizability](https://en.wikipedia.org/wiki/linearizability) of transactions, whereas the other only provides [serializability](https://en.wikipedia.org/wiki/Serializability).
 By relaxing the ordering constraint, the peak transaction throughput supported by the system scales horizontally with the number of nodes, but the transaction history is unavailable making the system harder to audit retroactively.
 Both architectures handle multiple geo-distributed datacenter outages with a [recovery time objective (RTO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Time_Objective) of under ten seconds and a [recovery point objective (RPO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Point_Objective) of zero.
 
 There are two UHS-based architectures as follows:
+
 1. "Atomizer" architecture
-    - Materializes a total ordering of all transactions settled by the system in a linear sequence of batches.
-    - Requires vertical scaling as peak transaction throughput is limited by the performance of a single system component.
-    - Maximum demonstrated throughput ~170K transactions per second.
-    - Geo-replicated latency <2 seconds.
-1. "Two-phase commit" architecture
-    - Transaction history is not materialized and only a relative ordering is assigned between directly related transactions.
-    - Combines [two-phase commit (2PC)](https://en.wikipedia.org/wiki/Two-phase_commit_protocol) and [conservative two-phase locking (C2PL)](https://en.wikipedia.org/wiki/Conservative_two-phase_locking) to create a system without a single bottlenecked component where peak transaction throughput scales horizontally with the number of nodes.
-    - Maximum demonstrated throughput ~1.7M transactions per second.
-    - Geo-replicated latency <1 second.
-
-Read the [2PC & Atomizer architecture guide](docs/uhs-architectures.md) for a detailed description of the system components and implementation of each architecture.
+   - Materializes a total ordering of all transactions settled by the system in a linear sequence of batches.
+   - Requires vertical scaling as peak transaction throughput is limited by the performance of a single system component.
+   - Maximum demonstrated throughput ~170K transactions per second.
+   - Geo-replicated latency <2 seconds.
+2. "Two-phase commit" architecture
```

Comment:
I believe we use 1. for all bullets in a numbered list intentionally

Reply:
I think a lot of these changes, including the reindentation, spacing and renumbering, are due to automated reformatting. I was under the impression these were going to be reverted or, at least, pulled into a standalone commit so they could be evaluated on their own merits.